env0/data_cost_credentials_test.go

```diff
@@ -97,14 +97,14 @@ func TestUnitAzureCostCredentialsDataSource(t *testing.T) {
 
```

Comment:
Left the test as is (with the exception of error messages), this shows that the data_cost_credentials isn't required and can safely be deleted.

Reply:
will refactor in next PR...